tools/steam-shell.pike

```diff
@@ -468,7 +468,7 @@ int list(string what)
     toappend = "Here is a list of all "+what+" in the current room\n";
   foreach(display,string str)
   {
-    a=a+(str+"    ");
+    a=a+(str+"\n");
     if(str=="Invalid command")
```

Comment:
No I have used the table formatting which accepts strings separated by new line.